sys/include/crypto/helper.h

```diff
@@ -10,7 +10,7 @@
  * @ingroup     sys_crypto
  * @{
  *
- * @file        helper.h
+ * @file        sys/include/crypto/helper.h
```

Comment:
The changes to this file are unrelated

Reply:
I believe `cpu/avr8_common/include/helper.h` file created some warning conflict with `sys/include/crypto/helper.h` at static checks and I decided fix it. It become complete irrelevant once `cpu/avr8_common/include/helper.h` won't be necessary.
Thank you for remind me.